Replicon.Cryptography.SCrypt/KeyDerivationFunction.cs

```diff
@@ -0,0 +1,311 @@
+﻿using System;
+using System.Security.Cryptography;
+
+namespace Replicon.Cryptography.SCrypt
+{
+    //FIXME: Use https://github.com/viniciuschiele/Scrypt as a dependency after PR #11 is released.
```

Comment:
Please add to this note that the below file is copied from that project, include the original author's name and a reference to the license under which we believe it is permitted to use this code.

Reply:
Done